app/src/pages/prompt/PromptVersionsList.tsx

```diff
@@ -81,8 +83,13 @@ export const PromptVersionsList = ({ prompt }: PromptVersionsListProps) => {
     prompt
   );
   return (
-    <View height="100%" overflow="scroll">
-      <Flex direction="column" width={300}>
+    <View
+      height="100%"
+      overflow="scroll"
+      width={PROMPT_VERSIONS_LIST_WIDTH}
+      minWidth={PROMPT_VERSIONS_LIST_WIDTH}
```

Comment:
This should likely be set to width=100% and have the parent constrain the width with a px value. That way we can render this list in both the prompt aside and the versions tab and let the parent decide how much space it takes up.